tasks/main.yml

```diff
@@ -133,6 +133,14 @@
        virtualenv_python=/usr/bin/python{{ python_version }}
```

Comment:
Also, the requirements seems to be a dictionary keys by the environment. Let's be consistent with other variables and just make this a variable, and the project can set it in the environment's variables if they want, or globally.